modules/Core/pages/api/v2/index.php

```diff
@@ -725,52 +734,45 @@ private function removeGroupFromDiscord() {
     }
     
     // Create a report
-    private function createReport(){
+    private function createReport() {
         // Ensure the API key is valid
         if($this->_validated === true) {
-            if(!isset($_POST) || empty($_POST)){
+            if(!isset($_POST) || empty($_POST)) {
                 $this->throwError(6, $this->_language->get('api', 'invalid_post_contents'));
             }
 
             // Validate post request
-            // Player UUID, report content, reported player UUID and reported player username required
-            if((!isset($_POST['reporter_uuid']) || empty($_POST['reporter_uuid']))
-                || (!isset($_POST['reported_username']) || empty($_POST['reported_username']))
-                || (!isset($_POST['reported_uuid']) || empty($_POST['reported_uuid']))
-                || (!isset($_POST['content']) || empty($_POST['content']))){
+            // Player id, report content, reported player id and reported player username required
+            if((!isset($_POST['reporter']) || empty($_POST['reporter']))
```

Comment:
Here we'll need 3 calls - one to get the reporter user ID, one to get the reported user ID, and then the third to this report endpoint

Reply:
@Derkades

Reply:
Reported user is a username not an ID. The reported user may not have an account

Reply:
Ah yeah my bad, fixing